app/make_mwa_incoh_beam.c

```diff
@@ -343,7 +335,7 @@ void read_step( VoltageContext *vcs_context, uint64_t gps_second,
                 gps_second,
                 1,
                 coarse_chan_idx,
-                data,
+                (char*)data,
```

Comment:
We just had a discussion in STDERR on Monday about how `char` can actually be interpreted as `unsigned char` or `signed char` depending on the target architecture and compiler. To avoid potential errors later on different systems, the general advice is to always explicitly say whether you want `signed` or `unsigned`.

I won't let this hold up the PR, but I'll let you (whoever is reading this) decide whether this should be made into an "issue" to be followed up later, in order to make this code as portable as possible.

Reply:
Cool. I have made it an issue, but like you say I won't let it hold this up.